.github/workflows/azure-static-web-apps-delightful-cliff-0acdaf71e.yml

```
@@ -15,29 +14,37 @@ jobs:
+    permissions:
+       id-token: write
+       contents: read
     steps:
       - uses: actions/checkout@v4
         with:
           submodules: true
           lfs: false
+      - name: Install OIDC Client from Core Package
+        run: npm install @actions/core@1.6.0 @actions/http-client
+      - name: Get Id Token
+        uses: actions/github-script@v6
```

actionlint (1.7.4)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)